app/views/people/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>List of People</h1>
+<div class="task-container">
+ <% @people.each do |person| %>
+ <%= link_to person.name, "/people/#{person.id}" %>
+ <% person.tasks.each do |task| %>
+   <% uncompleted_tasks = [] %>
+    <% if task.completion_status != "done" %>
+    <% uncompleted_tasks << task %>
+    <%= uncompleted_tasks.length %>
+    <% end %>
```

Comment:
This is great that you're thinking about displaying just the uncompleted tasks-- it would probably be better to put this logic in the controller or model and have an instance variable that you can use in the view.